utils/network.ts

```diff
@@ -42,6 +45,40 @@ export type GasInfo = {
 	l1Fee: number;
 };
 
+const loadInfuraProvider = (networkId: NetworkId) => {
+	if (!process.env.NEXT_PUBLIC_INFURA_PROJECT_ID) {
+		throw new Error('You must define NEXT_PUBLIC_INFURA_PROJECT_ID in your environment');
+	}
+
+	return loadProvider({
+		networkId,
+		infuraId: process.env.NEXT_PUBLIC_INFURA_PROJECT_ID,
+	});
+};
+
+const loadBlastProvider = (networkId: NetworkId) => {
+	const networkSlug = BLAST_NETWORK_LOOKUP[networkId];
```

Comment:
Should we add the same check as above on the env param existing and throw when missing?